src/cpp/src/make_tokenizer_stateful.hpp

```diff
@@ -76,6 +96,8 @@ class MakeVocabDecoderSatateful : public ov::pass::ModelPass {
 
 const std::string ADD_SPECIAL_TOKENS_VAR_ID = "add_special_tokens";
 const std::string SKIP_SPECIAL_TOKENS_VAR_ID = "skip_special_tokens";
+const std::string MAX_PAD_LENGTH_VAR_ID = "max_pad_length";
+const std::string MAX_TRUNCATION_LENGTH_VAR_ID = "max_pad_length";
```

Comment:
values should not defined in .hpp files, but only declared via `extern`. Definition should be moved to .cpp

Otherwise, if I include such header in other .cpp files, I will have multiple definitions when link final OpenVINO GenAI library.

Reply:
Done. Even better to use `inline`